package.json

```diff
@@ -32,19 +32,19 @@
   },
   "homepage": "https://github.com/mixpanel/domsuite#readme",
   "dependencies": {
+    "@babel/cli": "^7.17.6",
+    "@babel/preset-env": "^7.16.11",
+    "core-js": "^3.21.1",
```

Comment:
The babel deps should go back in devDependencies, while core-js stays here. The distinction is whether the code is included/executed at runtime. The babel deps are just for the build phase.

Also pin down the exact version of everything, no ^

Reply:
whops, totally messed up with dependencies and devDependencies. Just fixed~